cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -349,7 +349,7 @@ def test_slurm_validators_are_called_with_correct_argument(test_datadir, mocker)
     root_volume_encryption_consistency_validator.assert_has_calls(
         [call(encryption_settings=[("queue1", True), ("queue2", True)])]
     )
-    ebs_volume_type_size_validator.assert_has_calls([call(volume_type="gp3", volume_size=35)])
+    ebs_volume_type_size_validator.assert_has_calls([call(volume_type="gp3", volume_size=40)])
```

Comment:
This should've been changed in https://github.com/aws/aws-parallelcluster/pull/5592.